### ISSUE TITLE
Babel Information

### ISSUE THREAD

**Issue by Jukorop** [TF2CutContentWiki/TCRFDark#4]
Not sure whether this is intentional or not, but the information shown in the babel thing is still displayed in white unlike everything else.
![Снимок экрана_2022-04-26_10-16-44(1)](https://user-images.githubusercontent.com/75884396/165244452-abaf54b0-1713-462c-9a75-1013eb0afb97.png)

**Comment by TF2CutContentWiki** [comment]
Yeah the bg colors of the babel boxes were left alone. I can definitely tweak them to fit this theme a bit better.
